README.md

```diff
@@ -47,15 +39,51 @@ mvn install
 
 a. To run single test session.
 ```
-  mvn -Dexec.mainClass="com.browserstack.app.JavaSample" -Dexec.classpathScope=test test-compile exec:java
+mvn exec:exec -Dexec.executable="java" -Dexec.classpathScope=test -Dexec.args="-cp %classpath -javaagent:/path/to/.m2/repository/browserstack-java-sdk/jar com.browserstack.app.JavaSample"
+```
+
+b. To run local test session.
+- Ensure that you have set `browserstackLocal: true` in your `browserstack.yml` config.
+```
+mvn exec:exec -Dexec.executable="java" -Dexec.classpathScope=test -Dexec.args="-cp %classpath -javaagent:/path/to/.m2/repository/browserstack-java-sdk/jar com.browserstack.app.JavaLocalSample"
```

Comment:
Add a sample command, for windows and osx as mentioned in docs. 
Both for single test and local test session or let's document exec plugin command